TASK-2_1/Chatbot.cs

```diff
@@ -0,0 +1,61 @@
+using System.Text.RegularExpressions;
+
+namespace Task2_1;
+
+public class Chatbot
+{
+    public string UserName { get; set; }
+    public string UserSurname { get; set; }
+
+    public DateTime Date { get; set; }
+
+    public static string InputName(int nameType)
+    {
+        string name;
+        const string pattern = @"\d*\s*\W*";
+        const string target = "";
+        var regex = new Regex(pattern);
+
+        do
+        {
+            Console.Write("Введите Ваш" + (nameType == 1 ? "у фамилию: " : "е имя: "));
+            name = Console.ReadLine();
+            name = regex.Replace(name, target);
+        } while (name == ""); // проверка на пустую строку
```

Comment:
You already defined empty string for 'target' variable. So, instead of hardcoding - put it here

Reply:
What if I use "String.IsNullOrEmpty(name)" in line 24 instead? To make it more human readable. And if "target" value will never be changed, we can write "" directly in line 23: _regex.Replace(name, "")._